crates/core/component/funding/src/liquidity_tournament/action/mod.rs

```diff
@@ -34,6 +37,17 @@ pub struct LiquidityTournamentVoteBody {
     pub rk: VerificationKey<SpendAuth>,
 }
 
+impl LiquidityTournamentVoteBody {
+    /// Get the asset id that should be incentivized.
+    ///
+    /// This will return None if the denom is not a base denom.
+    pub fn incentivized_id(&self) -> Option<asset::Id> {
+        REGISTRY
+            .parse_denom(&self.incentivized.denom)
```

Comment:
I presume this still work for unknown denoms right?

Reply:
https://rustdoc.penumbra.zone/main/src/penumbra_sdk_asset/asset/registry.rs.html#70

Yeah, the case in which it returns None is only when it's a known non-base denom.